docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -1,57 +1,332 @@
-sure you select _ext-net_. Then, click the green _Create_ button to 
-create the new network.
+    ```bash
+    openstack network create nuremberg
+    ```
 
-![Finish creating network](assets/new-net-panel/shot-05.png)
+    By issuing the command above, you immediately get information
+    regarding the new network:
 
-In a few seconds, the new network will be readily available. You may 
-see all defined networks, in all supported regions, by selecting 
-_Networking_ > _Networks_ (see the left-hand side pane on the 
-{{extra.gui}}).
+    ```plain
+    +---------------------------+--------------------------------------+
+    | Field                     | Value                                |
+    +---------------------------+--------------------------------------+
+    | admin_state_up            | UP                                   |
+    | availability_zone_hints   |                                      |
+    | availability_zones        |                                      |
+    | created_at                | 2022-10-30T14:31:49Z                 |
+    | description               |                                      |
+    | dns_domain                | None                                 |
+    | id                        | 201d458b-9b47-4408-9736-980bec77d405 |
+    | ipv4_address_scope        | None                                 |
+    | ipv6_address_scope        | None                                 |
+    | is_default                | False                                |
+    | is_vlan_transparent       | None                                 |
+    | mtu                       | 1500                                 |
+    | name                      | nuremberg                            |
+    | port_security_enabled     | True                                 |
+    | project_id                | dfc700467396428bacba4376e72cc3e9     |
+    | provider:network_type     | None                                 |
+    | provider:physical_network | None                                 |
+    | provider:segmentation_id  | None                                 |
+    | qos_policy_id             | None                                 |
+    | revision_number           | 1                                    |
+    | router:external           | Internal                             |
+    | segments                  | None                                 |
+    | shared                    | False                                |
+    | status                    | ACTIVE                               |
+    | subnets                   |                                      |
+    | tags                      |                                      |
+    | updated_at                | 2022-10-30T14:31:50Z                 |
+    +---------------------------+--------------------------------------+
+    ```
 
-![All networks in all regions](assets/new-net-panel/shot-06.png)
+## Adding subnet, router, and finishing up
```

Comment:
"Finishing up" sounds a bit odd here. "Adding a subnet and router" would be better, I think.